src/lms-ds-loader/edfi_lms_ds_loader/helpers/argparser.py

```diff
@@ -153,7 +153,7 @@ def parse_main_arguments(args_in: List[str]) -> MainArguments:
 
     USE_INTEGRATED = "--useintegratedsecurity"
```

Comment:
I like having constants at the top 🤔

Reply:
Generally I do that when the constant is for re-usability. This case is more like when you're in TypeScript and you define a little variable to hold a string, making it easier to incorporate the string into a next step, and you realize that there's no reason for the string to change. So you define it as a `const` instead of a plain variable. I don't think we have that construct in Python, so I used uppercase to signal that this is intended for use as an immutable string.